game/views.py

```
@@ -1015,4 +1027,108 @@ def password_reset_account_selection(request):
+
+def confirm_delete_account(request, uidb64, token):
+
+    try:
+
+        uid = force_str(
+            urlsafe_base64_decode(uidb64)
+        )
+
+        user = User.objects.get(pk=uid)
+
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)